library/src/main/java/com/bumptech/glide/manager/SupportRequestManagerFragment.java

```diff
@@ -20,6 +21,9 @@
  * @see com.bumptech.glide.RequestManager
  */
 public class SupportRequestManagerFragment extends Fragment {
+
+    private static final String TAG = SupportRequestManagerFragment.class.getSimpleName();
```

Comment:
This will throw for anyone starting to use Glide `SupportRequestManagerFragment` is 29 long and the max length is 23, that's why Sam used "RMFragment", please mimic the fix in v4.